content-src/components/DiscoveryStreamComponents/DSImage/DSImage.jsx

```diff
@@ -42,7 +42,13 @@ export class DSImage extends React.PureComponent {
   }
 
   componentDidMount() {
-    this.observer = new IntersectionObserver(this.onSeen.bind(this));
+    this.observer = new IntersectionObserver(this.onSeen.bind(this), {
+      // Assume an image will be eventually seen if it is within 520px of the viewport
+      // This is half the average Desktop vertical screen size:
+      // http://gs.statcounter.com/screen-resolution-stats/desktop/north-america
+      rootMargin: `520px`,
```

Comment:
nit: why string literal

Reply:
I always use them because there's no reason to use any other string syntax anymore AFAIK.